src/devices/legrand.ts

```diff
@@ -385,7 +385,7 @@ const definitions: Definition[] = [
     },
     {
         zigbeeModel: [' Micromodule switch\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'],
-        model: '064888',
+        model: '064888 / 199142',
```

Comment:
Maybe add it to `whiteLabel` instead?

Reply:
Sorry for my late answer! I forgot this topic!
i did as suggested. this is much bettert! thanks